src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -277,6 +277,41 @@ try {
 }
 ```
 
+###### `preRenderedFetch`
+
+<p>
+
+**Type:** `(url: ErrorPagePath) => Promise<Response>`<br />
+**Default:** `fetch`<br />
+<Since v="5.6.0" />
+</p>
+
+In some environments, it would be useful to provide a custom implementation for fetching the prerendered error pages, for example if you don't have `fetch` available
+or cannot call the server from itself.
+
+The example below reads 500.html and 404.html from disk instead of performing an HTTP call.
```

Comment:
```suggestion
A function that allows you to provide custom implementations for fetching prerendered error pages.

This is used to override the default `fetch()` behavior, for example when `fetch()` is unavailable or when you cannot call the server from itself.

The following example reads 500.html and 404.html from disk instead of performing an HTTP call:
```